README.md

```diff
@@ -70,50 +82,70 @@ The default template uses JUnit 5. You may be used to JUnit 4, but here are some
-    ```shell
-    odo url create --port 9080
-    ```
-1. Once your project has been initialized and the URL created, you can push your application to your openshift cluster for the first time using the following command:
+1. Once your project has been initialized, you can push your application to your openshift cluster for the first time using the following command:
 
     ```shell
-    odo push 
+    odo push
     ```
 
     Upon a first time `odo push` invocation, the following occurs:
-    1. An Open Shift managed kubernetes pod is launched within the namespace created in step 3 above, 
-    1. A container based on the image referenced in the devfile is started in that pod 
-    1. odo syncs your local source code directory to that container. (under the `/projects` directory)
-    1. The app is compiled and built within the container and liberty dev mode is started to host the app once built.
-    1. The URL is activated and is usable to access the app 
+    - An Open Shift managed kubernetes pod is launched within the namespace created in step 3 above,
+    - A container based on the image referenced in the devfile is started in that pod
+    - odo syncs your local source code directory to that container. (under the `/projects` directory)
+    - The app is compiled and built within the container and liberty dev mode is started to host the app once built.
+    - A URL is activated and is usable to access the app
+
+1. (Optional) Create a secure URL
+
+    The Open Liberty stack activates a default HTTP route on port 9080. This can be used for most development and testing. If, however, a secure route is needed (e.g. accessing MP Metrics), you can create one with the following steps:
+
+    1. Create the secure URL
+
+        ```shell
+        odo url create --secure --port=9443
+        odo push
+        ```
+
+    1. (Temporary workaround) Manually edit the Route in OCP.
+
+       The "odo url create" command does not currently support additional Route configuration options. Additionally, devfiles does not support tls.termination configuration: https://github.com/eclipse/che/issues/14622
+
+       In order to properly configure the secure route do the following:
+
+       1. Navigate to your secure route in OCP (Networking -> Routes -> <route_name>)
+       1.  Remove `spec.tls.path: /` from the YAML configuration file
+       1. Manually set `spec.tls.termination: passthrough` in the YAML configuration file
+
+1. Retrieve the URL for your app using the odo CLI:
 
-1. Retrieve the URL created in step 5 using the odo CLI:
     ```shell
     odo url list
     ```
-    
+
     Edits to source code can be made using any preferred IDE (Eclipse, VSCode or others). To see source code changes reflected in the running application re-issue the `odo push` command after any edit session is complete and saved. Source code changes will be reflected in the running container within a few seconds.
```

Comment:
Hmm.. maybe just remove IDE for the time being.

Reply:
👍